cheatsheets/Vulnerability_Disclosure_Cheat_Sheet.md

```diff
@@ -23,7 +23,7 @@ It is recommended to use responsible disclosure when dealing with vulnerabilitie
 
 - alert the company, multiple times and persons if needed.
 - request [CVE Identification](http://cve.mitre.org/cve/request_id)([RedHat CVE HowTo](https://github.com/RedHatProductSecurity/CVE-HOWTO)).
-- alert trusted 3rd party like National CERT, Data Privacy regulator if apply. Eventually, some security researchers like [Brian Krebs](https://krebsonsecurity.com/) or [Troy Hunt](https://www.troyhunt.com/) (non-limitative. check your network first) can be intermediate too or provide support.
+- alert trusted 3rd party like National CERT, Data Privacy regulator if apply. Eventually, some security researchers like [Brian Krebs](https://krebsonsecurity.com/) or [Troy Hunt](https://www.troyhunt.com/) (non-exhaustive. check your network first) can be intermediate too or provide support.
```

Comment:
Yes, `limitative` is in the dictionary. No, it isn't really used. Again, this was probably French. But someone will have to consider whether my correction is a reasonable replacement.

Reply:
It's an improvement, but the whole sentence still doesn't read well.

TBH, I think this who CS needs an overhaul, so happy to merge this change for now.